Sources/TuistGenerator/Generator/ProjectGenerator.swift

```diff
@@ -134,16 +135,19 @@ final class ProjectGenerator: ProjectGenerating {
     }
 
     private func generatePbxproject(project: Project,
+                                    projectFileElements: ProjectFileElements,
                                     configurationList: XCConfigurationList,
                                     groups: ProjectGroups,
                                     pbxproj: PBXProj) throws -> PBXProject {
+        let defaultRegions = ["en"]
```

Comment:
is `en` a default region always, is there a possibility you can have other region as default?

Reply:
At the moment its set the match the default region generated by Tuist:

https://github.com/tuist/tuist/pull/792/files/18e490be0c7cbdda8e00e64c3d8514ab63d05c39#diff-8003e7a5fde9a1f00d9e93c3bad58745R148

Could be a separate enhancement to support specifying the default development regions.